ir/pass_manager.h

```diff
@@ -36,6 +36,7 @@ class PassManager : virtual public Visitor, virtual public Backtrack {
     void addPasses(const std::initializer_list<Visitor *> &init) {
         never_backtracks_cache = -1;
         for (auto p : init) if (p) passes.emplace_back(p); }
+    void deletePasses() { passes = vector<Visitor *>(); }
```

Comment:
where is it used?

Reply:
In the backend refactor. I would like to fold PHV passes into a PassManager, and because of PHV analysis and allocation are split before and after table placement, need a way to clear and re-populate passes.

Reply:
This sounds suspiciously like you're solving the wrong problem, though I'm prepared to be convinced otherwise. *Without giving details about Barefoot-internal code on this public repo*, could you please give an example of a thing you're trying to do that you can't do with the existing PassManager interface?